app/narwos.py

```diff
@@ -712,6 +712,9 @@ def get_exec_filtered_report():
     report_ext = ext_settings.get('ext', '.csv')
     report_mimetype = ext_settings.get('mimetype', 'text/csv')
 
+    if not PATH_TO_FILTERED_DF:
```

Comment:
thats another case, but that paths need validation at the beggining, so before app starts, just after laoding config file

Reply:
if we had time, we would implement it